create_images.sh

```diff
@@ -15,35 +15,35 @@ echo "==> Generating images for patch level $_os_patch_level"
 
 # mkdir -p $(dirname $_boot_out)
 
-if [ "$_has_dtbo" = "true" ]; then
-    _dts_folder=$(realpath $_out/arch/arm64/boot/dts/qcom)
-    _files=$(find $_dts_folder -iname "*.dtbo")
+if [[ "$_has_dtbo" == "true" ]]; then
+    _dts_folder=$(realpath "$_out"/arch/arm64/boot/dts/qcom)
```

Comment:
I personally find this more readable:

```suggestion
    _dts_folder=$(realpath "$_out/arch/arm64/boot/dts/qcom")
```

Would you mind applying that to all cases?